merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/AerieAppDriver.java

```diff
@@ -45,7 +45,8 @@ public static void main(final String[] args) {
     final var simulationAgent = ThreadedSimulationAgent.spawn(
         "simulation-agent",
         new SynchronousSimulationAgent(planController, missionModelController));
-    final var simulationController = new CachedSimulationService(stores.results(), simulationAgent);
+    final CachedSimulationService simulationController = new CachedSimulationService(stores.results(), simulationAgent);
```

Comment:
🔶 This change doesn't seem to do anything besides make the variable type explicit - we can probably revert it.

Reply:
Thanks for that catch. Failed to fully clean up after an experiment I backed out of :D